src/lib/core/backtrace.cc

```diff
@@ -344,7 +352,7 @@ __asm__ volatile(
 	"\tmov sp, x2\n"
 	/* Setup fisrst arg */
 	"\tmov x0, %0\n"
-	"\tbl %2\n"
+	"\tbl unw_getcontext_f\n"
```

Comment:
AFAICS from CI run for osx_arm64_11_2, this way of calling function from inside an asm block is not portable enough.
```
Undefined symbols for architecture arm64:
  "unw_getcontext_f", referenced from:
      coro_unwcontext(unw_context_t*, coro_context*) in libcore.a(backtrace.cc.o)
     (maybe you meant: _unw_getcontext_f)
```
You may use a different approach (see my comment below).